site2/website/blog/2021-08-xx-Apache-Pulsar-2-7-3.md

```diff
@@ -0,0 +1,139 @@
+---
+author: Bo Cong, Anonymitaet
+title: Apache Pulsar 2.7.3
+---
+
+# What’s New in Apache Pulsar 2.7.3
+
+The Apache Pulsar community releases version 2.7.3! 34 contributors provided improvements and bug fixes that delivered 79 commits.
+
+## Highlights
+
+- Cursor reads adhere to the dispatch byte rate limiter setting and no longer cause unexpected results. [PR-11249](https://github.com/apache/pulsar/pull/11249)
+
+- The ledger rollover scheduled task runs as expected. [PR-11226](https://github.com/apache/pulsar/pull/11226)
+
+This blog walks through the most noteworthy changes. For the complete list including all enhancements and bug fixes, check out the Pulsar 2.7.3 Release Notes.
```

Comment:
add a link for "Pulsar 2.7.3 Release Notes"?

Reply:
The Pulsar 2.7.3 Release Notes was merged but not shown on https://pulsar.apache.org/en/versions/ due to the issue of the Apache website. @congbobo184 has reported this issue to Apache. I will add the link once the issue gets fixed.